cmd/brew-test-bot.rb

```diff
@@ -373,6 +373,10 @@ def diff_formulae(start_revision, end_revision, path, filter)
       elsif travis_pr
         @url = "https://github.com/#{ENV["TRAVIS_REPO_SLUG"]}/pull/#{ENV["TRAVIS_PULL_REQUEST"]}"
         @hash = nil
+      elsif ENV["CI_PULL_REQUEST"] && !ENV["CI_PULL_REQUEST"].empty?
+        # Circle CI build of a pull request.
```

Comment:
`# Use Circle CI pull-request variables for pull request jobs.` and put it above `elsif ENV["CI_PULL_REQUEST"]`. Just to check: is this actually empty but set, ever?